pkg/volume/flexvolume/probe.go

```diff
@@ -228,6 +228,10 @@ func (prober *flexVolumeProber) updateEventsMap(eventDirAbs string, op volume.Pr
 // Each file or directory change triggers two events: one from the watch on itself, another from the watch
 // on its parent directory.
 func (prober *flexVolumeProber) addWatchRecursive(filename string) error {
+	// this may be called with an actual absolute path on Windows (with a C:\ prefix).
+	// But the prober.fs.Walk below will execute filepath.Join(fs.root, filenameAbove), which
+	// will result in an incorrect path, you can't join C:\path and C:\another\path.
+	filename = strings.TrimPrefix(filename, `C:\`)
```

Comment:
For reference, this can happen on this call: https://github.com/kubernetes/kubernetes/blob/909faa3a9b33161992af0684c5582353b5be6d9b/pkg/volume/flexvolume/probe.go#L176

Basically, ``filepath.Abs`` is used for getting the "absolute path" of the ``pluginDir`` and ``event.Name``. On Windows, for paths like ``/something``, ``filepath.Abs`` will return ``C:\\something``, which is then passed here in this function.